engine/lib/base_jobmaster.go

```diff
@@ -237,6 +240,11 @@ func (d *DefaultBaseJobMaster) CurrentEpoch() libModel.Epoch {
 	return d.master.currentEpoch.Load()
 }
 
+// OnOpenAPIInitialized implements BaseJobMaster.BaseJobMaster
```

Comment:
This logic looks strange.
`OnOpenAPIInitialized ` is the thing framework require user to implement.

Just call this `impl.OnOpenAPIInitialized()`. in  BaseJobMaster.Init() logic